docs/source/getting-started/tabular-data/grouping.ipynb

```diff
@@ -9,9 +9,9 @@
     "The OpenDP Library allows you to compute statistics over grouped data.\n",
     "We'll examine three settings with progressively more public information:\n",
     "\n",
-    "- Protected group keys: `public_info=None`\n",
-    "- Public group keys: `public_info=\"keys\"`\n",
-    "- Public group lengths: `public_info=\"lengths\"`\n",
+    "- Stable group keys\n",
+    "- Explicit group keys\n",
+    "- Invariant group keys\n",
```

Comment:
Is the `public_info` part no longer accurate or relevant? Or maybe it just doesn't belong in these titles? If someone is reading code, and trying to understand the semantics, I think it could be useful to highlight this part, but your call.

Reply:
It is still accurate and relevant, but I wanted to change how this was communicated for two reasons. 

1. I also feel that I should more strongly caution against using "public_info" and more clearly associate it with what is generally considered to be a more negative word "invariant".
2. Now that we have more options for privately releasing group keys (stable under zCDP and explicit), public_info is only specified when working with the third category (invariants).